sciplot/multiplot.hpp

```diff
@@ -53,6 +53,10 @@ class multiplot
     /// Construct a default multiplot object
     multiplot();
 
+    /// Toggle automatic cleaning of temporary files (enabled by default). Pass false if you want to keep your script / data files.
+    /// Call cleanup() to remove those files manually.
+    auto autoclean(bool enable = true) -> void;
```

Comment:
For the future, we could also have a `sciplot::autoclean(bool)` method so that we don't have to call `autoclean` for each instance..

Reply:
Or canvas::autoclean(bool)...